e2e/cypress/integration/team_settings/join_closed_team_with_not_allowed_email_spec.js

```diff
@@ -26,19 +27,84 @@ describe('Team Settings', () => {
                 Enable: false,
             },
         });
+    });
+
+    beforeEach(() => {
+        // # Login as sysadmin
+        cy.apiAdminLogin();
 
         cy.apiInitSetup().then(({team}) => {
+            testTeam = team;
             cy.visit(`/${team.name}`);
```

Comment:
why not do this in the `before` section?

Reply:
I want to have a new team setup each time so that the settings are clearly changed in each case.